code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -142,15 +142,33 @@ var/list/VVckey_edit = list("key", "ckey")
 		if(confirm != "Continue")
 			return
 
-	var/list/names = sortList(L)
+	var/assoc = 0
+	if(L.len > 0)
+		var/a = L[1]
+		if(!isnum(a) && L[a] != null)

```

Comment:
Thanks for the information, will update this one. Wish there was inbuilt function to check if something's hashable/whatever it uses for indexing.